terraform/providers.tf

```diff
@@ -17,3 +17,9 @@ provider "aws" {
   alias               = "us-west-2"
   region              = "us-west-2"
 }
+
+provider "google" {
```

Comment:
Since this provider is specific to the bitbar-devicepool project, it should probably be moved to the ' terraform/bitbar-devicepool/gcp-providers.tf'.  For each terraform GCP directory, we can just specify the provider each time.

Reply:
Done.